strawberry/field.py

```diff
@@ -39,6 +39,7 @@ def __init__(
         base_resolver: Optional[StrawberryResolver] = None,
         permission_classes: List[Type[BasePermission]] = (),  # type: ignore
         default_value: Any = undefined,
+        default_factory: Union[Callable, object] = undefined,
```

Comment:
can we use `UNSET` instead of undefined? 😊

Reply:
Sure. Should I change `default_value` as well?

Reply:
yes, then we can remove this: https://github.com/strawberry-graphql/strawberry/blob/44dd42d4f442eb77078c8def079b22ed4800c583/strawberry/types/type_resolver.py#L31-L32

but if it starts looking like a big change we can defer it to another PR :)